Bio/Phylo/_utils.py

```diff
@@ -127,12 +127,12 @@ def draw_graphviz(tree, label_func=str, prog='twopi', args='',
 
     Examples
     --------
-    >>> import pylab
-    >>> from Bio import Phylo
-    >>> tree = Phylo.read('ex/apaf.xml', 'phyloxml')
-    >>> Phylo.draw_graphviz(tree)
-    >>> pylab.show()
-    >>> pylab.savefig('apaf.png')
+    import pylab
+    from Bio import Phylo
+    tree = Phylo.read('PhyloXML/apaf.xml', 'phyloxml')
+    Phylo.draw_graphviz(tree)
+    pylab.show()
+    pylab.savefig('apaf.png')
```

Comment:
Does this need turning into a literal block? i.e. Intend a level, and add a double-colon ending line like this above it:

```
Load a PhyloXML format tree, and draw a PNG using GraphViz::
```

Reply:
yes, these examples that produce a picture/image on the screen should'n be on run_test, 
- some expect an interaction (to close the picture), that stops the test till you close the picture.
- some had left Mathplot hang, most likely haven't close the draw properly.
- we shouldn't get pictures on the screen when we run the test.